app/controllers/items_controller.rb

```diff
@@ -1,22 +1,51 @@
 class ItemsController < ApplicationController
+  before_action :set_item, only: [:show, :edit, :udpate, :destroy]
+
   def index
+    @items = Item.all
   end
 
   def show
   end
 
   def create
+    @item = Item.new(item_params)
+    if @item.save
+      redirect_to item_path(@item)
+    else
+      render 'new'
+    end
   end
 
   def new
+    @item = Item.new
   end
 
   def update
+    if @item.update(item_params)
+        redirect_to item_path(@item.id)
+    else
+      render 'edit'
+   end
   end
 
   def edit
   end
 
   def destroy
+    @item.destory
+    #redirect_to dash_board
```

Comment:
nice, we can address this tomorrow and it will also fix the avatar issue of directing the user to their dashboard